project/Dependencies.scala

```diff
@@ -51,6 +54,8 @@ object Dependencies {
     r2dbcSpi,
     r2dbcPool,
     r2dbcPostgres,
+    h2,
+    r2dbcH2,
```

Comment:
Yeah, I wrote a fixme about that somewhere. I thought it is best to keep Postgres as a regular transitive dependency for now, but maybe it is better to do it right away, when this will anyway end up in a minor release?

Reply:
h2 provided now